### PR TITLE
feat: remove axios in favor of fetch API

### PR DESCRIPTION
**Pull request by josefineschaefer**
BREAKING CHANGE: axios is no longer a dependency

### CODE REVIEW

File: lib/types.ts [hunk 1]
```diff
@@ -10,26 +10,26 @@ export interface SbSvelteSDKOptions extends SbSDKOptions {
...
+  // StoryblokCacheProvider,
   StoryblokClient,
   StoryblokComponentType,
-  StoryblokConfig,
-  StoryblokManagmentApiResult,
-  StoryblokResult,
+  ISbConfig,
+  ISbManagmentApiResult,
+  ISbResult,
   apiPlugin,
```
Comment (manuelschroederdev) [inline lib/types.ts:33]: I think `apiPlugin` should be a normal export

File: lib/index.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ export {
   useStoryblokBridge,
   apiPlugin,
   renderRichText,
-  RichTextSchema,
+  // RichTextSchema, // ISbSchema ?
```
[the remark this answers is not in the document]
Comment (alexjoverm) [reply lib/index.ts:9]: yes, along with `RichTextResolver` + `apiPlugin`

File: lib/types.ts (hunk 1 above)
Comment (josefineschaefer) [reply lib/types.ts:33]: good point, thank you

File: lib/index.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (josefineschaefer) [reply lib/index.ts:9]: done, thank you

File: lib/package.json [hunk 3]
```diff
@@ -28,7 +28,7 @@
     "prepublishOnly": "npm run build && cp ../README.md ./"
   },
   "dependencies": {
-    "@storyblok/js": "^2.0.5"
+    "@storyblok/js": "^2.0.6"
```
Comment (spences10) [inline lib/package.json:29]: Could this not be a dev dependency?

**Review by spences10 — APPROVED** [review]

**Comment by spences10** [comment]
Maybe try using [`vitePreprocess`] now for this and if that fails maybe can try adding in the build target to the `vite.config.js` file:

```js
build: {
  target: 'esnext',
},
```

[`vitePreprocess`]: https://kit.svelte.dev/docs/integrations#preprocessors

**Review by roberto-butti — APPROVED** [review]

**Review by spences10 — APPROVED** [review]

**Pull request merged**